jenkins/scripts/slurm_run.sh

```
@@ -56,6 +56,41 @@ env | sort
 
 echo "Full Command: $pytestCommand"
 
+if [[ "${TRTLLM_VISUAL_GEN_MULTINODE_SLURM_PARENT:-0}" == "1" ]]; then
+    # Scope the markers to the SLURM job: the stage's infra-retry loop can reuse
+    # jobWorkspace, and stale node_* files from a previous attempt would let
+    # rank 0 start before this attempt's nodes finish installing.
+    install_done_dir="${jobWorkspace}/visual_gen_multinode_install_done_${SLURM_JOB_ID:-0}"
```

Shellcheck (0.11.0)

[warning] jobWorkspace is referenced but not assigned.

(SC2154)